onnxruntime/core/providers/acl/activation/activations.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright(C) 2018 Intel Corporation
```

Comment:
Remove Intel?

Reply:
Thanks! This is a left over from the first ACL kernel implementation.